ctapipe/tools/tests/test_tools.py

```diff
@@ -19,10 +21,32 @@
 from pathlib import Path
 
 
+d = tempfile.TemporaryDirectory()
 GAMMA_TEST_LARGE = get_dataset_path("gamma_test_large.simtel.gz")
 LST_MUONS = get_dataset_path("lst_muons.simtel.zst")
 
 
+@pytest.fixture
```

Comment:
Might want to mark all of these new fixtures as `scope='module'` at least, or higher, since they get re-run every time you use the fixture in the default scope ('function'), which is very slow.

Reply:
Didn't know of that, thank you